### DISCUSSION TITLE
HJACK Azure application proxy

### DISCUSSION THREAD

**Discussion by Hemn1990** [hajkmap/Hajk#926] [Q&A] [answered]
Vi har en Hjack för intern använding på en windows server -
Vi vill publicera den via Application proxy men det verkar inte funka, några tips för hur man kan publicera denna ?

Mvh
A

**Comment by Hallbergs** [comment, marked as answer]
Halloj,

Jag har tyvärr ingen koll på Azure application proxy, (förhoppningsvis har någon annan information kring den), men installationsinstruktioner hittar du här: https://github.com/hajkmap/Hajk/wiki/Installation-guide-%28for-pre-packaged-releases%29

Henrik
